pkg/agent/plugin/workloadattestor/docker/docker.go

```diff
@@ -168,18 +172,52 @@ func (p *DockerPlugin) Configure(ctx context.Context, req *spi.ConfigureRequest)
 	if err != nil {
 		return nil, err
 	}
-	if config.CgroupPrefix == "" {
-		config.CgroupPrefix = defaultCgroupPrefix
+
+	if config.CgroupPrefix != "" || config.CgroupContainerIndex != nil {
+		if config.CgroupPrefix == "" || config.CgroupContainerIndex == nil {
+			return nil, errors.New("cgroup_prefix and cgroup_container_index must be specified together")
+		}
+
+		p.cgroupPrefix = config.CgroupPrefix
```

Comment:
Hmm, maybe we should log a warning message that these configurables are now deprecated and will be removed in a future release?